### PR TITLE
Feature toppage body

### PR DESCRIPTION
**Pull request by AyakaYasuda**

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -20,6 +36,11 @@ body {
 
 .santouka-img img {
   padding: 20px;
+  transition: .5s;
+}
+
+.santouka-img img:hover {
+    transform: scale(1.1,1.1);
```
Comment (SOunit) [inline main.css]: how about santouka-img-container if its parent of img?

Comment (SOunit) [reply main.css]: some use wrapper too. shorter than container.

**Review by SOunit — CHANGES_REQUESTED** [review]

File: main.css (hunk 1 above)
Comment (AyakaYasuda) [reply main.css]: I add -container to the class name.

File: index.html [hunk 2]
```diff
@@ -56,9 +56,21 @@
...
+        </div>
+      </div>
       <div class="main-container-Santouka">
-        <div class="santouka-img">
-          <img src="img/ramen3_shio_santouka.png" alt="Ramen from Santouka" />
+        <div class="santouka-img-container">
+          <img
+            src="img/ramen3_shio_santouka.png"
+            alt="Ramen from Santouka"
+            width="380"
```
Comment (SOunit) [inline index.html:71]: no styles inside html file!

**Review by SOunit — CHANGES_REQUESTED** [review]
no style inside html. style for css, structure for html. please never mix them.

**Pull request merged**